edisgo/data/import_data.py

```diff
@@ -2061,29 +2031,39 @@ def _retrieve_timeseries_from_oedb(config_data, weather_cell_ids):
         feedin = pd.read_sql_query(feedin_sqla.statement,
                                    session.bind,
                                    index_col=['source', 'w_id'])
+        return feedin
 
-        feedin.sort_index(axis=0, inplace=True)
+    if config_data['data_source']['oedb_data_source'] == 'model_draft':
+        orm_feedin_name = config_data['model_draft']['res_feedin_data']
+        orm_feedin = model_draft.__getattribute__(orm_feedin_name)
+        orm_feedin_version = 1 == 1
+    else:
+        orm_feedin_name = config_data['versioned']['res_feedin_data']
+        orm_feedin = supply.__getattribute__(orm_feedin_name)
+        orm_feedin_version = orm_feedin.version == config_data['versioned'][
+            'version']
 
-        timeindex = pd.date_range('1/1/2011', periods=8760, freq='H')
+    with session_scope() as session:
+        feedin = _retrieve_timeseries_from_oedb(session)
 
-        recasted_feedin_dict = {}
-        for type_w_id in feedin.index:
-            recasted_feedin_dict[type_w_id] = feedin.loc[
-                                              type_w_id, :].values[0]
+    feedin.sort_index(axis=0, inplace=True)
 
-        feedin = pd.DataFrame(recasted_feedin_dict, index=timeindex)
+    timeindex = pd.date_range('1/1/2011', periods=8760, freq='H')
```

Comment:
Do we want to hard-code the date '1/1/2011'?

Reply:
I didn't hard-code it, it's always been hard-coded. I also think it's fine that it is hard-coded since it's in the function that retrieves the pre-calculated feed-in time series from the oedb and they are valid for the year 2011.